Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -467,7 +467,7 @@
     <!-- YMSI: onboarding(tracing) - dialog about manual checking -->
     <string name="onboarding_manual_required_dialog_body">"Beachten Sie, dass Sie ohne Aktivierung der priorisierten Hintergrundaktivität die App mindestens alle 24 Stunden einmal aktiv aufrufen müssen, um Ihren Risikostatus aktualisieren zu können.\n\nSie können die priorisierte Hintergrundaktivität jederzeit in Ihren Einstellungen aktivieren."</string>
     <!-- XBUT: onboarding(tracing) - dialog about manual checking button -->
-    <string name="onboarding_manual_required_dialog_button">"Ok"</string>
+    <string name="onboarding_manual_required_dialog_button">"OK"</string>
```

Comment:
Note that most buttons automatically capitalize the label, and should  usually not be capitalized within the resource files as it reduces their re-usability.

Reply:
Ok, I will change it.
Just thought of consistency of the other "OK"-texts. I will check them too.